packages/graph/hash_graph/lib/graph/src/types/schema/entity_type/mod.rs

```diff
@@ -1,11 +1,15 @@
+mod links;
+
 use std::collections::{HashMap, HashSet};
 
 use error_stack::{ensure, Result};
+// I do not know why rustfmt insists to put this import here despite the fact that it's not an
+// external crate
+use links::{Links, ValueOrMaybeOrderedArray};
```

Comment:
I have not found a way to enforce this. The suggested changes require `rustfmt` to be run afterward.

```suggestion
use self::links::{Links, ValueOrMaybeOrderedArray};
```